src/test/scala/akka/persistence/cassandra/CassandraLifecycle.scala

```diff
@@ -76,7 +76,8 @@ trait CassandraLifecycle extends BeforeAndAfterAll { this: TestKitBase with Suit
       cassandraDirectory,
       configResource = cassandraConfigResource,
       clean = true,
-      port = 0
+      port = 0,
+      fork = true
```

Comment:
This is not a public API, it's just for Akka persistence itself. I set fork to true here both to test the feature, and also because it means you can now run the tests in more environments than just sbt (for example, if you try to run the tests in IntelliJ with this set to false, the first passes, and every subsequent one fails).